xbmc/interfaces/legacy/ListItem.cpp

```diff
@@ -390,8 +390,6 @@ namespace XBMCAddon
             item->GetVideoInfoTag()->m_strProductionCode = value;
           else if (key == "aired")
             item->GetVideoInfoTag()->m_firstAired.SetFromDateString(value);
-          else if (key == "credits")
-            item->GetVideoInfoTag()->m_writingCredits = StringUtils::Split(value, g_advancedSettings.m_videoItemSeparator);
```

Comment:
Hehe sure. I will drop that commit then and stop caring about removing silly API flaws. (Btw, from a certain perspective I would even call the possibility to be able to use both "credits" and "writer" for the same thing a bug because it is not transparent to the addon authors that one overwrites the other)